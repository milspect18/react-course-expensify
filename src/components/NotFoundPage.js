import React from 'react';
import { Link } from 'react-router-dom';

const NotFoundPage = () => (
    <div>
        That aint here man...
        <Link to='/' >Go Home</Link>
    </div>
);

export default NotFoundPage;
