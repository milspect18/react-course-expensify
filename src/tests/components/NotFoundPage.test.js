import { shallow } from 'enzyme';
import React from 'react';
import NotFoundPage from '../../components/NotFoundPage';

test('Should render the not found (404) page', () => {
    const wrapper = shallow(<NotFoundPage />);

    expect(wrapper).toMatchSnapshot();
});
