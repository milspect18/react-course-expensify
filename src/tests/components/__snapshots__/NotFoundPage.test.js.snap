// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the not found (404) page 1`] = `
<div>
  That aint here man...
  <Link
    to="/"
  >
    Go Home
  </Link>
</div>
`;
